rpm/sapwmp.spec

```diff
@@ -61,7 +61,13 @@ Configuration and utilities for collecting SAP processes under control group to
 %build
 ./autogen.sh
 %configure
+# make_build not defined in SLE12, so need conditions to check
```

Comment:
The [wiki](https://en.opensuse.org/openSUSE:Specfile_guidelines#Parallel_make) says that %make_build should be available in >= SLE12 SP3. We really don't want to go back that much into the past.
If the wiki's wrong and even SLE12 SP5 needs this, I'm fine with the change.

Reply:
In reality, this macro is not existed in SLE12SP5... rpm-4.11.2-16.21.1.x86_64

```
# rpmbuild --showrc |grep make
      echo "Running automake --gnu ${am_opt} ..."; 
      %{__automake} --add-missing --gnu ${am_opt}; 
-14: __automake	automake
-14: __make	/usr/bin/make
	    yes "" | make oldconfig 
	    make dep 
-14: make_install	make install DESTDIR=%{?buildroot}
-14: makeinstall	make DESTDIR=%{?buildroot:%{buildroot}} install
-14: perl_make_install	make DESTDIR=$RPM_BUILD_ROOT install_vendor
```

Reply:
Ack.